builder/config.go

```diff
@@ -199,3 +199,20 @@ func (config *MixConfig) validate() error {
 
 	return nil
 }
+
+// Print print variables and values of a MixConfig struct
+func (config *MixConfig) Print() {
+	rt := reflect.TypeOf(*config)
+	rv := reflect.ValueOf(*config)
```

Comment:
it was originally called rc here as well, but since this is the only place I needed rt, I changed it. I think it is better to just change the other to rv instead.